apps/condo/package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "@2gis/mapgl": "1.28.0",
-    "@apollo/client": "3.3.20",
+    "@apollo/client": "3.6.4",
```

Comment:
It seems, like this version is not compatible yet, because in `/property` page I'm getting following error:
> Invariant Violation: Could not find "client" in the context or passed in as an option. Wrap the root component in an <ApolloProvider>, or pass an ApolloClient instance in via options.

Reply:
Added task DOMA-3021 for that

Reply:
we need to check `[v3.5.10]`